.jshintrc

```diff
@@ -5,9 +5,9 @@
 "eqnull" : true,
 "indent": 2,
 "undef": true,
-"unused": true,
-"quotmark": "single",
-"maxlen": 90,
+"unused": false,
+"quotmark": "true",
+"maxlen": 110,
```

Comment:
Interesting, I am ok to keep the line limit at 110 characters then. Can you please open an issue in strong-docs to keep track of this problem? /cc @ritch